Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3.6-alpine
```

3.6 is the *minimum* version the bot runs on, however it is not the *only* version it runs on. I'd suggest using the latest version of python 3 rather than hardcoding it to an older version.
```suggestion
FROM python:3-alpine
```